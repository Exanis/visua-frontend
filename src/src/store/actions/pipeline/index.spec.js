import * as Actions from './index';

describe('Pipeline Actions', () => {
    it('getPipelineList', () => {
        expect(Actions.getPipelineList(1, 'test')).toMatchSnapshot();
    });
    it('onPipelineList', () => {
        expect(Actions.onPipelineList(['first', 'second'])).toMatchSnapshot();
    });
    it('createPipeline', () => {
        expect(Actions.createPipeline('test-name', 1, 'test')).toMatchSnapshot();
    });
    it('onPipelineCreateSuccess', () => {
        expect(Actions.onPipelineCreateSuccess()).toMatchSnapshot();
    });
    it('onPipelineCreateError', () => {
        expect(Actions.onPipelineCreateError(['error'])).toMatchSnapshot();
    });
    it('resetPipelineCreateState', () => {
        expect(Actions.resetPipelineCreateState()).toMatchSnapshot();
    });
});