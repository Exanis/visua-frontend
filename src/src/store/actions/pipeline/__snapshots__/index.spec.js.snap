// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pipeline Actions createPipeline 1`] = `
Object {
  "name": "test-name",
  "page": 1,
  "search": "test",
  "type": "pipeline.create",
}
`;

exports[`Pipeline Actions getPipelineList 1`] = `
Object {
  "page": 1,
  "search": "test",
  "type": "pipeline.list",
}
`;

exports[`Pipeline Actions onPipelineCreateError 1`] = `
Object {
  "error": Array [
    "error",
  ],
  "type": "pipeline.create.error",
}
`;

exports[`Pipeline Actions onPipelineCreateSuccess 1`] = `
Object {
  "type": "pipeline.create.success",
}
`;

exports[`Pipeline Actions onPipelineList 1`] = `
Object {
  "pipelines": Array [
    "first",
    "second",
  ],
  "type": "pipeline.onList",
}
`;

exports[`Pipeline Actions resetPipelineCreateState 1`] = `
Object {
  "type": "pipeline.create.reset",
}
`;
